hack/install-gometalinter.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Any reason not to use the gometalinter provided install.sh? https://github.com/alecthomas/gometalinter/blob/master/scripts/install.sh

Reply:
Or at least wrap it?

Reply:
Not at all - just didn't realize that it existed.  Will remove this file and point the instructions at the gometalinter version.